MegaSolution/CreatingLineFromRandomPartsOfOthers/NewLineCreation.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace CreatingLineFromRandomPartsOfOthers
+{
+  class NewLineCreator
+  {
+    public string CreateNewLine(string[] line)
```

Comment:
bad name, no comments, no decomposition, no single responsibility, one class for one method is bad